helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -41,9 +41,9 @@ storageClass:
     fsType: xfs
     volumeDescription: "Volume created by the HPE CSI Driver for Kubernetes"
     accessProtocol: "iscsi"
-    # Provisioning type for 3PAR primera
+    # Provisioning type for 3PAR primera (tpvv, tdvv, full)
     provisioningType: "tpvv"
-    # CPG for 3PAR primera
+    # Existing CPG name for 3PAR primera
     cpg: "FC_r6"
```

Comment:
Why are these parameters leaking out here when they have CSP defaults and are CSP specific?

Reply:
removed